app/models/customer.py

```diff
@@ -3,7 +3,8 @@
 class Customer(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String)
```

Comment:
Something to consider: do we want to require all columns by setting nullable=False? It looks like we would potentially need to recreate our tables if we did (https://stackoverflow.com/questions/33192062/how-does-nullable-false-work-in-sqlalchemy), but wanted to consider this.